src/services/reaction.service.js

```diff
@@ -1,65 +1,37 @@
 /* eslint-disable import/prefer-default-export */
 import models from '../db/models';
 
-const { Article, Reaction } = models;
+const { ArticleReaction } = models;
 /**
- * @method articleService
+ * @method articleReaction
```

Comment:
I think you mean reaction service

Reply:
Thanks for pointing that out. I will fix this